src/Controller/QuizTemplateController.php

```diff
@@ -73,9 +82,16 @@ public function showQuizzes(Request $request, array $requestAttributes): Respons
         $quizName = $this->quizTemplateService->getFromParameter('quizName', $request, "");
         $userId = $this->quizTemplateService->getFromParameter('userId', $request, "");
         $currentPage = (int)$this->quizTemplateService->getFromParameter('page', $request, 1);
-        $totalResults = (int)$this->quizTemplateService->getEntityNumberOfPagesByField(QuizTemplate::class, ['name' => $quizName, 'user_id' => $userId]);
-        $quizzes = $this->quizTemplateService->getEntitiesByField(QuizTemplate::class, ['name' => $quizName, 'user_id' => $userId], $currentPage, self::RESULTS_PER_PAGE);
-        $users = $this->quizTemplateService->getEntitiesByField(User::class, ['role' => 'Admin'], 1, 99999999);
+        $filters = ['name' => $quizName, 'user_id' => $userId];
+        $orderBy = $this->quizTemplateService->getFromParameter('orderBy', $request, "");
+        $sortType = $this->quizTemplateService->getFromParameter('sort', $request, "");
+        $totalResults = (int)$this->quizTemplateService->getEntityNumberOfPagesByField(QuizTemplate::class, $filters);
```

Comment:
I think this can be moved to a separate method called `buildFilter` or similar

Reply:
+1